experiments/ocean_forced_by_tides.jl

```diff
@@ -0,0 +1,73 @@
+
+using ClimaOcean
+using Oceananigans
+using OrthogonalSphericalShellGrids
+using Oceananigans.Units
+using Printf
+
+import Oceananigans.BuoyancyFormulations: ρ′ 
+import ClimaOcean.OceanSeaIceModels: reference_density, heat_capacity
+
+struct ZeroBuoyancy{R}
+    reference_density :: R
+end
+
+@inline ρ′(i, j, k, grid, ::ZeroBuoyancy, T, S) = zero(grid)
+@inline reference_density(r::ZeroBuoyancy) = r.reference_density
+@inline heat_capacity(r::ZeroBuoyancy) = 3990.0
+
+# all passive tracers
+equation_of_state = ZeroBuoyancy(1026.0)
```

Comment:
interesting! but there won't be internal tides with no stratification!

Reply:
right, this is a small test case that I have eventually removed from the PR, but kept in the comments for documentation purposes